e2e/cli-e2e/mocks/fs.mock.ts

```diff
@@ -4,7 +4,7 @@ import { ensureDirectoryExists } from '@code-pushup/utils';
 
 // @TODO move into testing library
 export function cleanFolder<T extends object>(
-  dirName = 'tmp',
+  dirName = 'tmp/cli-e2e',
```

Comment:
Why is this here? If it avoids testing problems in the file system I suggest we go with --persist.filename instead

Reply:
Verdaccio, by default, saves all data in the tmp directory. But when the test is starting, tmp is already removed. Changing the default dirName is avoiding it.

Reply:
> But when the test is starting, tmp is already removed. 

Could it be fixed by adopting `global-setup.ts` ? If not we should maybe document it a bit as I would not have thought about this reason.